test/unit/specs/components/Personalization/dom-actions/setHtml.spec.js

```diff
@@ -49,13 +49,16 @@ describe("Personalization::actions::setHtml", () => {
     const settings = {
       selector: "#setHtml",
       prehidingSelector: "#setHtml",
-      content: "<script>window.someEvar123 = 1;</script>",
+      content: "<script id='evar123'>window.someEvar123 = 1;</script>",
       meta
     };
     const event = { elements };
 
     return setHtml(settings, event).then(() => {
       expect(window.someEvar123).toEqual(1);
+
+      const scriptElements = document.querySelectorAll("#evar123");
+      expect(scriptElements.length).toEqual(1);
     });
```

Comment:
Can you add a test to see if an event handler defined in the inlineScript still works? And what if you have a setTimeout in the inline script?